app/assets/javascripts/controllers/container_live_dashboard/container_live_dashboard_controller.js

```diff
@@ -185,59 +187,66 @@ miqHttpInject(angular.module('containerLiveDashboard', ['ui.bootstrap', 'pattern
         yData.unshift(metric_id);
 
         // TODO: Use time buckets
-        $scope.chartData.xData = xData;
-        $scope.chartData['yData'+n] = yData;
+        dash.chartData.xData = xData;
+        dash.chartData['yData'+n] = yData;
 
-        $scope.chartDataInit = true;
-        $scope.loadCount++;
-        if ($scope.loadCount >= $scope.selectedItems.length) {
-          $scope.loadingData = false;
+        dash.chartDataInit = true;
+        dash.loadCount++;
+        if (dash.loadCount >= dash.selectedItems.length) {
+          dash.loadingData = false;
         }
       });
     };
 
-    $scope.refresh_graph_data = function() {
-      $scope.loadCount = 0;
-      $scope.loadingData = true;
-      $scope.chartData = {};
+    dash.refresh_graph_data = function() {
+      dash.loadCount = 0;
+      dash.loadingData = true;
+      dash.chartData = {};
 
-      $scope.selectedItems = $scope.items.filter(function(item) { return item.selected });
+      dash.selectedItems = dash.items.filter(function(item) { return item.selected });
 
-      for (var i = 0; i < $scope.selectedItems.length; i++) {
-        var metric_id = $scope.selectedItems[i].id;
-        $scope.refresh_graph(metric_id, i);
+      for (var i = 0; i < dash.selectedItems.length; i++) {
+        var metric_id = dash.selectedItems[i].id;
+        dash.refresh_graph(metric_id, i);
       }
     };
 
-    $scope.timeRanges = [
+    dash.getTenants = function(include) {
+      dash.tenantChanged = true;
```

Comment:
Doesn't quite make sense to set `tenantChanged` here (while downloading a list of options), does it? What about watching `dash.tenant` for changes instead?